cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -284,7 +454,7 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
           |For example, to show only b-tree indexes, use `SHOW BTREE INDEXES`.
           |Another more flexible way of filtering the output is to use the `WHERE` clause.
           |An example is to only show indexes not belonging to constraints.""".stripMargin,
-      prepare = _ => executePreparationQueries(List("create index for ()-[r:KNOWS]-() on (r.since)")),
+      prepare = _ => executePreparationQueries(List("create btree index for ()-[r:KNOWS]-() on (r.since)")),
```

Comment:
```suggestion
      prepare = _ => executePreparationQueries(List("create b-tree index for ()-[r:KNOWS]-() on (r.since)")),
```
?

Reply:
no this is a query run in the background, lets keep the syntax in the query (even if we use lower case instead of upper in this particular instance, it is not visible anyway)